.ci/test-e2e.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname $0)/.."
+
+version="$(cat ./VERSION)"
+
+mkdir -p /tm
+/cc/utils/cli.py config attribute --cfg-type kubernetes --cfg-name testmachinery --key kubeconfig > /tm/kubeconfig
+/testrunner run \
+    --tm-kubeconfig-path=/tm/kubeconfig \
+    --timeout=1800 \
+    --testruns-chart-path=.ci/testruns/default-testrun \
```

Comment:
I think the path needs to be a fs path so it would be probably: `.ci/testruns/default` ?

Reply:
Good catch!